src/compiler/program.ts

```diff
@@ -1290,12 +1287,16 @@ namespace ts {
                 }
             }
 
-            function collectRequireCalls(node: Node): void {
+            function collectDynamicImportOrRequireCalls(node: Node): void {

```

Comment:
I almost wish we tracked these during parse to avoid having to walk the entire source tree again, though there are issues with that when considering incremental parse scenarios. Not a requirement for this PR though.